third_party/netcdf-cxx.BUILD

```diff
@@ -0,0 +1,17 @@
+'''
+netcdf-cxx@4.3.1
+'''
+
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+cmake(
+    name = "netcdf-cxx",
```

Comment:
Do we need to provide `deps` here for the c-lib? Is it still a WIP?

Reply:
Yeah my bad. Idk how but a everything builds just fine even though I didn't specify the deps. I'll try to look into it.